server/clients/manageRecallsApiClient.ts

```diff
@@ -24,7 +24,7 @@ import { GenerateDocumentRequest } from '../@types/manage-recalls-api/models/Gen
 import { FieldAuditEntry } from '../@types/manage-recalls-api/models/FieldAuditEntry'
 import { FieldAuditSummary } from '../@types/manage-recalls-api/models/FieldAuditSummary'
 
-export async function prisonerByNomsNumber(nomsNumber: string, token: string): Promise<Prisoner | null> {
+export async function getPrisonerByNomsNumber(nomsNumber: string, token: string): Promise<Prisoner | null> {
```

Comment:
They're not necessarily a prisoner at this point, they could be 'not in custody'. Could be less confusing to use `getPersonByNomsNumber` ?

Reply:
sure, happy to name as you wish

Reply:
The class that is returned is called a "Prisoner" tho, rather than a Person. And comes from the /prisoner endpoint via the prisoner search api.

Reply:
fair enough.. just thinking of how a new dev will make sense of the codebase

Reply:
As a "new" dev to the UI, I found Person confusing. As to me its a very general term that covers all people, prisoner/offender seem more specific and are helpful as to what type of 'person' you are finding.